_includes/author-profile.html

```diff
@@ -208,7 +200,7 @@ <h3 class="author__name" itemprop="name">{{ author.name }}</h3>
               <i class="fab fa-fw fa-youtube" aria-hidden="true"></i> YouTube
             </a>
           </li>
-        {% else author.youtube %}
```

Comment:
This is a side fix that I did when I scanned through the file. Correct me if I'm wrong.

Reply:
Might not need the `assign` anymore either. Looks like `author.home` is only being used the one time so adding `relative_url` directly to that should be fine.

```
{% if author.home %}
  <a href="{{ author.home | relative_url }}">
  ...
```